omnipaxos/src/sequence_paxos/mod.rs

```diff
@@ -217,16 +217,16 @@ where
                 // Resend AcceptStopSign or StopSign's decide
                 if *phase == Phase::Accept {
                     if let Some(ss) = self.internal_storage.get_stopsign() {
+                        let decided_idx = self.internal_storage.get_decided_idx();
                         for follower in self.leader_state.get_promised_followers() {
-                            let decided_idx = self.internal_storage.get_decided_idx();
-                            if self.leader_state.get_accepted_idx(follower)
+                            if self.internal_storage.stopsign_is_decided() {
+                                self.send_decide(follower, decided_idx, true);
+                            }
+                            else if self.leader_state.get_accepted_idx(follower)
```

Comment:
If we are resending a Decide for a StopSign then our sequence number has progressed past when we sent the AcceptStopSign. Since we don't cache the AcceptStopSign's sequence number if we resent it we would just trigger a resync.

Now if a follower is missing the accept and decide we still trigger a resync when we resend the decide but send less messages (and dont send an AcceptStopSign with a false sequence number).